imageroot/events/module-removed/00event_validation

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env python3
```

Comment:
You can create a link to `module-add/00event_validation` instead of replicate the code

Reply:
done